geronimo-javamail_1.6/tck.adoc

```diff
@@ -41,20 +41,28 @@ Copy Geronimo Javamail 1.6 jar to the TCK folder
 
 `export CLASSPATH=$JARPATH/geronimo-javamail_1.6_mail-1.0.0-SNAPSHOT.jar`
 
-Edit $TCKHOME/lib/ts.jts:
+Edit `$TCKHOME/lib/ts.jte`:
 
+```
+JAVA_HOME=/home/jgallimore/.sdkman/candidates/java/8
 TS_HOME=/home/jgallimore/dev/ee8tck/mail-tck
 JARPATH=/home/jgallimore/dev/ee8tck/mail-tck
 LOCAL_CLASSES=$JARPATH/geronimo-javamail_1.6_mail-1.0.0-SNAPSHOT.jar
+```
 
 # Setup the mailbox
 
 ```
-cd tests/mailbox
-java fpopulate -s test1 -d imap://user01%40james.local:1234@localhost:1143/test1
+cd tests/mailboxes/
+wget https://github.com/javaee/javamail/releases/download/JAVAMAIL-1_6_2/javax.mail.jar
+unzip javax.mail.jar
+java -cp . fpopulate -s test1 -d imap://user01%40james.local:1234@localhost:1143/test1
```

Comment:
the -cp . was need it in my laptop, otherwise I got a `Could not find or load main class fpopulate`.
I also had to use the javax.mail.jar instead of the geronimo one.